src/__tests__/App.test.js

```diff
@@ -19,6 +31,48 @@ describe('App', () => {
     await fireEvent.click(queryByText(/pomodoro/i));
     expect(tomatoButton.textContent).toMatch(/25:00/);
   });
+  it('pomodoro timer shows 20:00 when 5 minutes have passed', async () => {
+    jest.useFakeTimers();
+
+    render(App);
+
+    const tomatoBtn = queryByTestId('tomato-button');
+    const fiveMinutesInMs = 5 * 60 * 1000;
+
+    expect(tomatoBtn.textContent).toMatch(/25:00/);
+
+    await fireEvent.click(tomatoBtn);
+    await jest.advanceTimersByTime(fiveMinutesInMs);
+    expect(tomatoBtn.textContent).toMatch(/20:00/);
+
+    jest.runOnlyPendingTimers();
+    jest.useRealTimers();
+  });
+
+  it('pomodoro timer shows 0 seconds left when 25 minutes have passed', async () => {
+    console.log = jest.fn();
+    jest.useFakeTimers();
+
+    render(App);
+
+    const tomatoBtn = queryByTestId('tomato-button');
+    const twentyFiveMinutesInMs = 25 * 60 * 1000;
+
+    expect(tomatoBtn.textContent).toMatch(/25:00/);
+
+    await fireEvent.click(tomatoBtn);
+    await jest.advanceTimersByTime(twentyFiveMinutesInMs);
+    expect(tomatoBtn.textContent).toMatch(/festa! 0:00/i);
+
+    // A sound should be played once the timer has finished.
+    // We use the AudioMock at the beginning of this test file to log when a sound 
+    // is played because there is no way to actually play the sound during a test. 
+    expect(console.log.mock.calls[0][0]).toMatch(/playing sound/i);
```

Comment:
```suggestion
  expect(audio.play).toBeCalledTimes(1);
```

If we pass `audio` in, we can expect on `audio.play` directly: `render(App, { audio }); 👍 

Let's also `expect(audio.play).not.toBeCalled()` after the click but before advancing timers.